### ISSUE TITLE
jumpToItem in ListView doesn't work properly if called just after creation.

### ISSUE THREAD

**Issue by sbrednikhin** [cocos2d/cocos2d-x#14510]
In ListView::doLayout size was wrong at the moment of layout changing. I caused wrong item position and wrong shift in jumpToItem.

Possible fix:
Was:

```
void ListView::doLayout()
{
...
    _innerContainer->forceDoLayout();
    updateInnerContainerSize();
    _innerContainerDoLayoutDirty = false;
}
```

Must be:

```
void ListView::doLayout()
{
...
    updateInnerContainerSize();
    _innerContainer->forceDoLayout();
    _innerContainerDoLayoutDirty = false;
}
```

**Comment by zilongshanren** [comment]
@neokim  @liamcindy 
please take a look at this issue.

**Comment by neokim** [comment]
@sbrednikhin 
The issue is resolved in 603d788fbffe0add7dd8c7bdf59f46315dccb785 and already merged in the latest branch. The fix is exactly same as your recommendation. Thanks.
